src/bounds.story.jsx

```diff
@@ -1,8 +1,15 @@
 
 import React from 'react';
-
+import Stripe from './Stripe';
 import Bounds from './Bounds';
 import { storiesOf } from '@kadira/storybook';
 
 storiesOf('Bounds', module)
-	.add('default', () => <Bounds />);
+	.add('default', () => (
+		<div style={{width: '100%'}}>
+			<Stripe><Bounds>
+				<h3 className='text--display2'>Bounds</h3>
+				<p>Bounds is used as a non-visual content container that manages content measure and centers children.</p>
+			</Bounds></Stripe>
```

Comment:
I'd recommend against over-specifying each Story - they should ideally contain only a single component so that it's clear what style comes from the component rather than parent components. You can make a standalone reference example story that has lots of nested components, but the basic story should be pure.